crates/miden-objects/src/lib.rs

```diff
@@ -13,6 +13,8 @@ pub mod block;
 pub mod note;
 pub mod transaction;
 
+pub mod address;
+
```

Comment:
Nit: Any reason not to remove the newline and move this up to the other modules? If we do, auto-format will alphabetically order them.